cypress/e2e/ui/Automation/Embedded-Automate/Explorer/class.cy.js

```diff
@@ -1,36 +1,14 @@
 /* eslint-disable no-undef */
 
 describe('Automation > Embedded Automate > Explorer', () => {
-  before(() => {
-    // Create a Domain and Namespace before all the tests
-    cy.login();
-    cy.intercept('POST', '/ops/accordion_select?id=rbac_accord').as('accordion');
-    cy.menu('Automation', 'Embedded Automate', 'Explorer');
-    cy.get('#explorer_title_text');
-
-    // Creates a Domain
-    cy.get('[title="Datastore"]').click();
-    cy.get('[title="Configuration"]').click();
-    cy.get('[title="Add a New Domain"]').click();
-    cy.get('[name="name"]').type('TestDomain', {force: true});
-    cy.get('[name="description"]').type('This is a test domain');
-    cy.get('#enabled').check();
-    cy.get('[class="bx--btn bx--btn--primary"]').contains('Add').click();
-
-    // Check for the success message
-    cy.get('div.alert.alert-success.alert-dismissable').should('exist').and('contain', 'Automate Domain "TestDomain" was added').find('button.close').should('exist');
-
-    // Creates a Namespace
-    cy.get('[title="Datastore"]').click();
-    cy.get('[title="Automate Domain: TestDomain"]').click(); // Click on Domain
-    cy.get('[title="Configuration"]').click();
-    cy.get('[title="Add a New Namespace"]').click();
-    cy.get('[name="name"]').type('TestNameSpace', {force: true});
-    cy.get('[name="description"]').type('This is a test NameSpace');
-    cy.get('.bx--btn--primary').contains('Add').click();
-
-    // Wait for namespace to be visible
-    cy.get('[title="Automate Namespace: TestNameSpace"]', {timeout: 1000}).should('be.visible')
+  beforeEach(() => {
+    cy.appFactories([
+        ['create', 'miq_ae_domain', {name: 'TestDomain'}],
+    ]).then((results) => {
+      cy.appFactories([
+        ['create', 'miq_ae_namespace', {name: 'TestNameSpace', domain_id: results[0].id}]
+    ])
+    });
```

Comment:
This is how you can use factories in cypress to create the domain and namespace and replaces all of that code on the left hand side.  If you're testing the creation in the UI, the code on the left is fine but since this is just setup code, the factories above should be used.